source/bisect.ts

```diff
@@ -0,0 +1,54 @@
+import {OptionsSync} from 'webext-options-sync-per-domain';
+
+import {RGHOptions, perDomainOptions} from './options-storage';
+
+export default async function bisectFeatures(): Promise<void> {
+	const options = perDomainOptions.getOptionsForOrigin();
+	const featuresState = Object.fromEntries(Object.entries(await options.getAll()).filter(([key]) => key.startsWith('feature:')));
+	const savedFeaturesState = {...featuresState};
+
+	// Test with all features disabled first
+	await onlyEnableFeatures(options, featuresState, []);
+	if (await confirmBug()) {
+		window.alert('It’s a native GitHub bug!');
```

Comment:
This isn't 100% true as not all features can be disabled.

Reply:
We could change the message to something like
```text
The bug comes from a permanent CSS feature or GitHub itself.
```